README.md

```diff
@@ -1,6 +1,222 @@
+  - [Accessibility](#accessibility)
+  - [Session history, navigation, and bfcache](#session-history-navigation-and-bfcache)
+- [Summary of differences between portals and iframes](#summary-of-differences-between-portals-and-iframes)
+- [Alternatives considered](#alternatives-considered)
+- [Security and privacy considerations](#security-and-privacy-considerations)
+- [Stakeholder feedback](#stakeholder-feedback)
+- [Acknowledgments](#acknowledgments)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Example
+
+A document can include a `<portal>` element, which renders the specified URL in a portal context:
+
+```html
+<portal id="myPortal" src="https://www.example.com/"></portal>
+```
+
+This is somewhat like an iframe, in that it provides a rendering of the specified document at `https://example.com/`. However, a portal is much more restricted than an iframe: user interaction with it [does not pass through to the portaled document](#interactivity), and when the portaled document is cross-origin, capabilities like storage and `postMessage()` communication are [prohibited](#privacy-threat-model-and-restrictions) while rendering in the portal context. This allows portals to be used as a more-restricted view onto another document when the full capabilities of an iframe are not needed.
+
+In exchange for these restrictions, portals gain an additional capability that iframes do not have. A portal can be *activated*, which causes the embedding window to navigate, replacing its document with the portal:
+
+```js
+myPortal.activate();
+```
+
+_Note: [issue #174](https://github.com/WICG/portals/issues/174) discusses making this the default behavior when clicking on a portal, similar to a link._
+
+At this point, the user will observe that their browser has navigated to `https://www.example.com/` (e.g., via changes to the URL bar contents and back/forward UI). Since `https://example.com/` was already loaded in the portal context, this navigation will occur seamlessly and instantly, without a network round-trip or document re-initialization.
+
+For more advanced use cases, the `https://www.example.com/` document can react to activation, using the `portalactivate` event. It can use this event to adapt itself to its new context, and even to adopt its *predecessor* (the document which previously occupied the tab) into a new portal context.
+
+```js
+window.addEventListener('portalactivate', e => {
+  document.body.classList.add('displayed-fully');
+  document.requestStorageAccess().then(() => {
+    document.getElementById('user').textContent = localStorage.getItem('current-user');
+  });
+
+  let predecessor = e.adoptPredecessor(document);
+  console.assert(predecessor instanceof HTMLPortalElement);
+  document.body.appendChild(predecessor);
+});
+```
+
+## Use cases
+
+_See the ["Key Scenarios"](./key-scenarios.md) document for more detail on each of these, including visualizations._
```

Comment:
This contradicts the warning at the top of `key-scenarios.md` a bit, but it's probably worth keeping anyway.